core/space/services/services.go

```diff
@@ -7,9 +7,11 @@ import (
 
 	"github.com/FleekHQ/space-daemon/core/textile/hub"
 	"github.com/FleekHQ/space-daemon/core/vault"
+	"golang.org/x/sync/errgroup"
 
 	"github.com/FleekHQ/space-daemon/config"
 	"github.com/FleekHQ/space-daemon/core/env"
+	node "github.com/FleekHQ/space-daemon/core/ipfs/node"
```

Comment:
```suggestion
 "github.com/FleekHQ/space-daemon/core/ipfs/node"
```
The extra `node` at the beginning is redundant, you can remove it

Reply:
That's not compiling. When I removed the "node" at the front, it complains on `*node.IpfsNode` that node is undefined.  Is that because `node` itself is not a package? I.e., it's package name is still `ipfs`.  Should it be it's own package? @maurycy  Not sure if I'm missing some go mod detail.